GabrielaRamirez/EjerciciosHackerrank/src/basics/IfElse.java

```diff
@@ -0,0 +1,29 @@
+package basics;
+import java.util.Scanner;
+
+public class IfElse {
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        int N = scanner.nextInt();
```

Comment:
variables name N ??
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names

Reply:
same, forgot you told us we needed to change the name variables